client/src/pages/OfferDash/index.tsx

```diff
@@ -120,6 +122,14 @@ export default function OfferDash () {
     let formWidth: number = mobile ? 175: 225;
     let margins: number = mobile ? 2 : 7;
 
+    if(error) {
+        <>
```

Comment:
do return <> ... </>

Reply:
Sorry I'm not sure what you mean for this

Reply:
Like the following 
```
if(error) {
    return {
        <>
           <Loading open={isLoadingCycle} />
            <SideDrawer />
            <Error />
        </>
    }
 }
```